test/blink/2-keyframes-with-offsets.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
-<script src="testharness/testharness.js"></script>
-<script src="testharness/testharnessreport.js"></script>
+<script src="../web-platform-tests/resources/testharness.js"></script>
```

Comment:
Note that having local resource folders is used in web-platform-tests as well, not just Blink imported tests. I don't see why using relative paths should be confusing, adding extra indirection via config files is much harder to read to me.